site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -50,6 +50,32 @@ public function editGradeableRequest($gradeable_id, $nav_tab = 0) {
         'electronic_bulk' => 'TA/Instructor will (bulk) upload scanned .pdf for online manual grading'
     ];
 
+    /**
+     * Creates a gradeable based on uploaded JSON data
+     * @return JsonResponse
+     * @AccessControl(role="INSTRUCTOR")
+     * @Route("/api/upload/{_semester}/{_course}", methods={"POST"})
+     * @Route("/courses/upload/{_semester}/{_course}", methods={"POST"})
```

Comment:
On all of our other routes we have `/courses/{_semester}/{_course}/...`. Is there a reason for it to be different that the others? I think it's better to keep it consistent.